### PR TITLE
Remove varying parts from box tests report

### PR DESCRIPTION
**Pull request by krzema12**
Values like timings or test run-dependent paths make the diff very noisy. For now, I just commented them out since they may become useful one day. I didn't invest time into making them e.g. a CLI option it's just a quick change with high impact.

# Testing

Before:
```
... Unresolved reference: kotlin.test (78,56) in /tmp/box.testsProject_pythonTest_12424857009541523426/testRootGW2/unitTest_exceptionTableSplitNoReturn/js-tests/JS_TESTS/2.kt  0.039
```

After:

```
... Unresolved reference: kotlin.test (78,56) in <path-truncated>  0.039
```

### CODE REVIEW

**Comment by krzema12** [comment]
Box tesrs report comparison fails because of difference in including detailed reasons. One file has exception messages, and the other one doesn't. TODO: investigate.

**Comment by krzema12** [comment]
@SerVB could you try running it on your machine in a spare moment?

File: python/experiments/generate-box-tests-reports.main.kts [hunk 1]
```diff
@@ -52,15 +52,23 @@ fun List<TestResult>.writeSummaryTsvToFile(targetFile: Path) = targetFile.toFile
...
-            pythonExecutionTime ?: "",
+            failureDetailedReason?.removeVaryingPaths() ?: "",
+//            kotlinCompilationTime ?: "",
+//            pythonExecutionTime ?: "",
         ).joinToString("\t"))
     }
 }
 
+fun String.removeVaryingPaths() =
+    // The exception message is sometimes provided locally and not provided locally, so trying to normalize it below.
```
Comment (krzema12) [inline python/experiments/generate-box-tests-reports.main.kts]: "provided locally and not provided **on CI**"

File: python/experiments/generate-box-tests-reports.main.kts [hunk 2]
```diff
@@ -52,15 +52,23 @@ fun List<TestResult>.writeSummaryTsvToFile(targetFile: Path) = targetFile.toFile
...
-            kotlinCompilationTime ?: "",
-            pythonExecutionTime ?: "",
+            failureDetailedReason?.removeVaryingPaths() ?: "",
+//            kotlinCompilationTime ?: "",
+//            pythonExecutionTime ?: "",
         ).joinToString("\t"))
     }
 }
 
+fun String.removeVaryingPaths() =
```
Comment (krzema12) [inline python/experiments/generate-box-tests-reports.main.kts]: Paths -> Parts

**Review by SerVB — APPROVED** [review]

**Pull request merged**